codecov.yml

```diff
@@ -1,4 +1,5 @@
 ignore:
+  - "pennylane/devices/default_mixed.py"
```

Comment:
I'd prefer not to update this file, as I didn't even know it existed, so we probably won't remember to come back here. 

If we are having issues with the `execute` method, we can just do:
```
def execute(self, circuits, execution_config=None):
    raise NotImplementedError
```

And then
```
with pytest.raises(NotImplementedError):
     dev.execute(QuantumScript())
```